core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/extension/elytron/main/module.xml

```diff
@@ -31,7 +31,6 @@
         <exclude path="org/wildfly/extension/elytron/ElytronExtension"/>
         <exclude path="org/wildfly/extension/elytron/ElytronSubsystemTransformers"/>
         <exclude path="org/wildfly/extension/elytron/_private"/>
-        <exclude path="org/wildfly/extension/elytron/capabilities/_private"/>
```

Comment:
Just to check, why was this removed?

Reply:
Ah, probably because that doesn't exist anymore.

Reply:
This is the `elytron` module.

Reply:
Yeah, I'm a bit confused by what you mean. Those files were [moved to elytron-common](https://github.com/cam-rod/wildfly-core/tree/27350b0f43fbb9efa590769e042048a2fe48c191/elytron/common/src/main/java/org/wildfly/extension/elytron_common/capabilities/_private), so I removed the exclusion from ` elytron`.

Reply:
Ah, I think when I was looking before, it looked like things were moved from `src/main/java/org/wildfly/extension/elytron/capabilities/_private` to `src/main/java/org/wildfly/extension/elytron` so was wondering why the corresponding path in the `elytron` module wasn't updated but perhaps I missed that things were actually moved to `elytron-common` instead?

Reply:
I did move around some of those folders after the first review, so it might have changed a bit